<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>public_debugDialog</class>
 <widget class="QDialog" name="public_debugDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>按住拖动</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>refresh</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>New Column</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>New Column</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>New Column</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>New Column</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>New Column</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item row="0" column="4">
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item row="1" column="3">
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item row="1" column="4">
      <property name="text">
       <string>1</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
